src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Logging/ProjectOutputWindowProjectLogger.cs

```diff
@@ -33,6 +33,7 @@ public bool IsEnabled
         {
             get { return _options.IsProjectOutputPaneEnabled; }
```

Comment:
We've hit and miss on it this. I personally do not like it - but if the team agrees, we can apply it consistently.